buildpacks/leiningen/buildpack.toml

```diff
@@ -0,0 +1,21 @@
+api = "0.9"
+
+[buildpack]
+id = "heroku/leiningen"
+version = "3.2.0"
+name = "Leiningen"
+clear-env = true
+homepage = "https://github.com/heroku/buildpacks-jvm"
+description = "Official Heroku buildpack for leiningen applications."
```

Comment:
Updating the Docker Hub descriptions reminded me of this - we'll need to update the descriptions/keywords etc here to use the new style introduced in #603.